src/transxchange/read.rs

```diff
@@ -681,10 +716,15 @@ fn load_routes_vehicle_journeys_calendars(
             .try_only_child("DepartureTime")?
             .text()
             .parse());
+        let vehicle_journey_timing_links = vehicle_journey
+            .children()
+            .filter(|child| child.name() == "VehicleJourneyTimingLink")
+            .collect::<Vec<&Element>>();
```

Comment:
Tip: You might be able to just do the following.
```suggestion
            .collect::<Vec<_>>();
```
`.coellct()` mostly need to know in which type of Collection object it needs to gather the items but from the iterator, it already knows what is the items' type.